### DISCUSSION TITLE
An Issue while getting Biometric data from Zkteco machines.

### DISCUSSION THREAD

**Discussion by sharukhkhan234** [horilla-opensource/horilla#207] [General]
I am facing some problems regarding getting Biometric data from Zkteco machines. It hides the data when an employee or user by mistake does check-out instead of check-in, or check-out instead of check-in, we are facing this problem with an employee with a night shift. Sometimes it fetches data and sometimes no record is found. But actually, these records are present in the machines, and can't be found in the software, How to update these records? or any solution.

**Comment by horilla-opensource** [comment]
Hi @sharukhkhan234 ,
That is because the user might have done two consecutive checkouts. Without a check-in, the user cannot have a checkout record in Horilla.

**Reply by sharukhkhan234** [reply]
Ok, thank you!
